core/src/main/java/edu/wpi/grip/core/sources/NetworkTableEntrySource.java

```diff
@@ -0,0 +1,147 @@
+package edu.wpi.grip.core.sources;
+
+import edu.wpi.grip.core.Source;
+import edu.wpi.grip.core.events.SourceHasPendingUpdateEvent;
+import edu.wpi.grip.core.events.SourceRemovedEvent;
+import edu.wpi.grip.core.operations.network.MapNetworkReceiverFactory;
+import edu.wpi.grip.core.operations.network.NetworkReceiver;
+import edu.wpi.grip.core.sockets.OutputSocket;
+import edu.wpi.grip.core.sockets.SocketHint;
+import edu.wpi.grip.core.sockets.SocketHints;
+import edu.wpi.grip.core.util.ExceptionWitness;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.assistedinject.AssistedInject;
+import com.google.inject.name.Named;
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Provides a way to get a {@link Types Type} from a NetworkTable that GRIP is connected to.
+ */
+@XStreamAlias("grip:NetworkValue")
```

Comment:
Should make this more explicit that it's a NetworkTable value. HTTP or ROS inputs may be added in the future.